### PR TITLE
Content production template

### PR DESCRIPTION
**Pull request by marksantoso**

### CODE REVIEW

File: content/governing-content/understand-content-lifecycle/content-production-template/main-content.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: templates/table
+table:
+  -
+    - |
+        Main content
+  -
+    - |
+    The first sentence sets an expectation for what’s on the page
```
Comment (dominikwilkowski) [inline content/governing-content/understand-content-lifecycle/content-production-template/main-content.md]: to make it clearer for the content authors I would indent this to be consistent

File: src/sass/components/tables.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  td {
+	  p:last-child {
+		  margin-bottom: 0px;
+	  }
+  }
+
+  td,
+  th {
+    padding: uikit-space( 0.5 );
```
Comment (dominikwilkowski) [inline src/sass/components/tables.scss]: oh boy indentation ... :)

File: src/templates/table.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+				</div>
+			</div>
+		</div>
+	);
+};
+
+
+Table.propTypes = {
+	/**
+	 * table: tables key values
```
Comment (dominikwilkowski) [inline src/templates/table.js]: you wanna add the yaml here for a good example table so the docs show it.

File: content/governing-content/understand-content-lifecycle/content-production-template/main-content.md (hunk 1 above)
Comment (marksantoso) [reply content/governing-content/understand-content-lifecycle/content-production-template/main-content.md]: done please review

**Pull request merged**